hs_access_control/models.py

```diff
@@ -2285,6 +2426,29 @@ def get_resources_with_explicit_access(self, this_privilege):
 
             return BaseResource.objects.filter(Q(pk__in=view) | Q(pk__in=immutable)).distinct()
 
+    def get_users_with_explicit_access(self, this_privilege):
+        """
+            Get a list of users for which the group has the specified privilege
+
+            :param this_privilege: one of the PrivilegeCodes
+            :return: QuerySet of user objects (QuerySet)
+        """
+
+        if __debug__:
+            assert this_privilege >= PrivilegeCodes.OWNER and this_privilege <= PrivilegeCodes.VIEW
+
+        if this_privilege == PrivilegeCodes.OWNER:
+            return self.owners
+        elif this_privilege == PrivilegeCodes.CHANGE:
+            return User.objects.filter(is_active=True,
+                                       u2ugp__group=self.group,
+                                       u2ugp__privilege=PrivilegeCodes.CHANGE)
+
+        else:  # this_privilege == PrivilegeCodes.ViEW
+            return User.objects.filter(is_active=True,
+                                       u2ugp__group=self.group,
+                                       u2ugp__privilege=PrivilegeCodes.VIEW)
+
```

Comment:
@alvacouch @pkdash I got the point of the assertions helping in debugging code, however, in this specific case, the code logic assumes the input parameter ```this_privilege``` is in valid range, and thus in production where assert has no effect, if by any chance an invalid input parameter is passed in, it will be treated in the code logic to have ```VIEW``` privilege, and returns all users with view privileges. I feel it is better in the code logic to return None instead for an invalid input privilege number rather than assuming the input parameter will be in a valid range, and if it is not Owner, and not Change, then it is View. We may never run into an invalid input parameter to this function anyway, so it is not that important, but I think it is still better from code logic perspective to handle it that way.